runtime/parachains/src/inclusion_inherent.rs

```diff
@@ -127,6 +129,47 @@ decl_module! {
 	}
 }
 
+/// Limit the number of backed candidates processed in order to stay within block weight limits.
+///
+/// Use a configured assumption about the weight required to process a backed candidate and the
+/// current block weight as of the execution of this function to ensure that we don't overload
+/// the block with candidate processing.
+///
+/// If the backed candidates exceed the available block weight remaining, then skips all of them.
+/// This is somewhat less desirable than attempting to fit some of them, but is more fair in the
+/// even that we can't trust the provisioner to provide a fair / random ordering of candidates.
+fn limit_backed_candidates<T: Config>(
+	backed_candidates: Vec<BackedCandidate<T::Hash>>,
+) -> Vec<BackedCandidate<T::Hash>> {
+	const BACKED_CANDIDATE_WEIGHT_ASSUMPTION: usize = 10_000;
+
+	let block_weight_remaining = <T as frame_system::Config>::MaximumBlockWeight::get()
+		.saturating_sub(frame_system::Module::<T>::block_weight().total());
```

Comment:
the definition of weight has slightly changed since https://github.com/paritytech/polkadot/pull/1420 can you merge polkadot master ?

Reply:
Good catch, thanks! 5bc1f4f5.